docs/guides/insar_product_guide.md

```diff
@@ -191,18 +191,24 @@ All of the phase differences in a wrapped interferograms lie between -π and π.
 Before the interferogram can be unwrapped, it must be filtered to remove noise. This is accomplished using an adaptive spectral filtering algorithm. This adaptive interferogram filtering aims to reduce phase noise, increase the accuracy of the interferometric phase, and reduce the number of interferogram residues as an aid to phase unwrapping. In this case, residues are points in the interferogram where the sum of the phase differences between pixels around a closed path is not 0.0, which indicates a jump in phase.
 
 #### Masking
-Another step before unwrapping is to create a validity mask to guide the phase unwrapping process. This mask is generated from the coherence and amplitude (backscatter intensity) values. 
+Another step before unwrapping is to create a validity mask to guide the phase unwrapping process. This mask is generated by applying thresholds to the coherence and/or amplitude (backscatter intensity) values for an image pair. For On Demand InSAR products, we only use a coherence threshold.
 
-Coherence is estimated from the normalized interferogram and the co-registered intensity images using an MLI estimator with rectangular weighting with a 5x5 moving window. The pixel values in this file range from 0.0 (total decorrelation) to 1.0 (perfectly coherent). Any input pixel with a coherence value less than 0.1 or an intensity value less than 0.2 is given a validity mask value of zero and not used during unwrapping.
+Coherence is estimated from the normalized interferogram using an MLI estimator with rectangular weighting with a 5x5 moving window. The pixel values in this file range from 0.0 (total decorrelation) to 1.0 (perfectly coherent). Any input pixel with a coherence value less than 0.1 is given a validity mask value of zero and not used during unwrapping.
 
-When the water masking option is applied, the validity mask is further amended to apply 0 values to any pixels classified as water in the water mask. In some cases, pixels over water may still meet the coherence and amplitude threshold criteria for inclusion, even though they are not valid for use during phase unwrapping. When processing scenes with extensive coverage by coastal waters or large inland waterbodies, there can be erroneous phase jumps introduced if unwrapping proceeds over water as if it were land. In such cases, choosing the option to apply the water mask can significantly improve the results. 
+!!! important "Change to Validity Mask Thresholds"
+
+    In the past, we also used a power threshold of 0.2 when generating the validity mask. While this approach tends to mask out inland waters, providing a less noisy interferogram in some cases, it also masks arid regions that have low amplitude values but reasonably high coherence. As of March 2022, we have set the power threshold to 0.0, so that coherence is the only driver of the validity mask.
```

Comment:
When we calculate the validity mask, we use  execute(f"rascc_mask {ifgname}.adf.cc {mmli} {width} 1 1 0 1 1 0.10 0.0 ", uselogging=True). mmli is the amplitude or power? Is the intensity a power? need consistent through out the doc.

Reply:
@talogan would have to verify what the mmli is. The amplitude product that we include in the output product is in power scale, and I assumed that the input for the amplitude threshold would also be in power scale. 

In this case, amplitude is used as the generic term for backscatter intensity, not to indicate the amplitude scale. I had understood all of these calculations to be using the power scale as input, but @talogan would be able to say for sure.